MinimedKit/PumpEvents/ChangeTimePumpEvent.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-public class ChangeTimePumpEvent: PumpEvent {
+public class ChangeTimePumpEvent: TimestampedPumpEvent {
```

Comment:
You're right, they should be; I think it might only be UnabsorbedInsulinPumpEvent that doesn't have a timestamp?